src/foam/nanos/auth/HumanNameTrait.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.INTERFACE({
+  package: 'foam.nanos.auth',
+  name: 'HumanNameTrait',
+
+  properties: [
+    {
+      class: 'String',
+      name: 'firstName',
+      tableWidth: 160,
+      documentation: 'First name of user.',
+      validateObj: function(firstName) {
+        if ( firstName.length > 70 ) {
+          return 'First name cannot exceed 70 characters.';
+        }
+
+        if ( /\d/.test(firstName) ) {
+          return 'First name cannot contain numbers.';
+        }
+      }
+    },
+    {
+      class: 'String',
+      name: 'middleName',
+      documentation: 'Middle name of user.',
+      validateObj: function(middleName) {
+        if ( middleName.length > 70 ) {
+          return 'Middle name cannot exceed 70 characters.';
+        }
+
+        if ( /\d/.test(middleName) ) {
```

Comment:
Reminds me: we should add a 'pattern' property to Property.

Reply:
Agree, although I disagree with the decision to disallow numbers in people's names.  Who are we to say that's valid or invalid.  I'm sure somebody somewhere has a digit in their name and I really don't see the value this extra "validation" is providing us.